styles/prosilver/template/kb_cat_body.html

```diff
@@ -96,9 +96,9 @@
 					<dd>
 						<ul class="post-buttons">
 						<!-- IF S_CAN_MOVE-->				<li id="move"><a href="#"	title="{L_MOVE_DRAGNDROP}"			class="button button-icon-only"><i class="icon fa-arrows fa-fw" aria-hidden="true" style="color: #00801D;"></i></i><span class="sr-only">{L_MOVE}</span></a></li></li><!-- ENDIF -->
-						<!-- IF not art_row.S_APPROVED -->	<li><a href="{art_row.U_ARTICLE}"	title="{L_NEED_APPROOVE}"	class="button button-icon-only"><i class="icon fa-check fa-fw" aria-hidden="true"></i><span class="sr-only">{L_NEED_APPROOVE}</span></a></li><!-- ENDIF -->
 						<!-- IF art_row.S_CAN_DELETE -->	<li><a href="{art_row.U_DELETE}"	title="{L_DELETE}"			class="button button-icon-only"><i class="icon fa-times fa-fw" aria-hidden="true" style="color: #AA0000;"></i></i><span class="sr-only">{L_DELETE}</span></a></li><!-- ENDIF -->
 						<!-- IF art_row.S_CAN_EDIT -->		<li><a href="{art_row.U_EDIT_ART}"	title="{L_EDIT}"			class="button button-icon-only"><i class="icon fa-pencil fa-fw" aria-hidden="true"></i><span class="sr-only">{L_EDIT}</span></a></li><!-- ENDIF -->
+						<!-- IF not art_row.S_APPROVED -->	<li><a href="{art_row.U_ARTICLE}"	title="{L_NEED_APPROOVE}"	class="button button-icon-only"><i class="icon fa-check fa-fw" aria-hidden="true"></i><span class="sr-only">{L_NEED_APPROOVE}</span></a></li><!-- ENDIF -->
```

Comment:
@AlexSheer, with this change, all MOVE, DELETE and EDIT button are aligned vertically, and if there are some APPROVE butons displayed these are at the end aligne also vertically.